packages/core-mobile/app/utils/units/fees.ts

```diff
@@ -1,13 +1,13 @@
 export function feeDenominationToBigint(
   fee: string,
-  isBtcNetwork: boolean
+  isBaseUnitRate: boolean
 ): bigint {
-  return isBtcNetwork ? BigInt(fee) : BigInt(fee) * BigInt(1e9)
+  return isBaseUnitRate ? BigInt(fee) : BigInt(fee) * BigInt(1e9)
```

Comment:
looks like we can just reuse nanoToWei() and weiToNano() here and remove both feeDenominationToBigint and bigIntToFeeDenomination